plugin/trino-hive/src/test/java/io/trino/plugin/hive/parquet/TestHdfsParquetDataSource.java

```diff
@@ -69,7 +69,6 @@ public void testPlanReadOrdering(DataSize maxBufferSize)
     public Object[][] testPlanReadOrderingProvider()
     {
         return new Object[][] {
-                {DataSize.ofBytes(0)}, // All large ranges
```

Comment:
We can use `1` here instead of removing the test case.

Reply:
the test asserts specific ranges to be read (0-200, 400-500, 700-900). 1 would imply different ranges (multiple ranges, 1 byte each)